gemrb/GUIScripts/InventoryCommon.py

```diff
@@ -253,6 +271,96 @@ def GetItemDescription (item, itemtype):
 	text = searchRE.sub(replacement, text)
 	return text
 
+def GetIWD2RestrictedAlignmentsAndStances(mask):
+	global AlignmentStancesTable
+	if not AlignmentStancesTable:
+		AlignmentStancesTable = GemRB.LoadTable("aligstan")
+
+	n = AlignmentStancesTable.GetRowCount()
+	showAlignments = mask & 0xE000
+	showStances = mask & 0x31000
+
+	text = ""
+	for i in range(n):
+		rowName = AlignmentStancesTable.GetRowName(i)
+		isStance = AlignmentStancesTable.GetValue(rowName, "STANCE")
+		useMask = AlignmentStancesTable.GetValue(rowName, "USABILITY")
+		nameRef = AlignmentStancesTable.GetValue(rowName, "NAME_REF")
+
+		if ((isStance and showStances) or (not isStance and showAlignments)) and useMask & mask:
```

Comment:
Maybe I'm being dumb, but why do you need to distinguish alignment axes at all? They will be checked in table order, so alignments will be properly grouped already.

Reply:
Yeah, serves no need any more.